content/app-dev/create-actor/run.md

```diff
@@ -1,50 +1,70 @@
+
+### Build the actor
+
+Building the actor is as easy as running `make` in the project's root directory. This will compile the actor and sign it with a freshly generated seed key. The `Makefile` that comes with the newly generated project lists claims for the actor including `wasmcloud:httpserver`, so it can work with the HTTP Server capability provider.
 
 ### Launch the Actor
 
-There are countless ways to run the actor we just created, but the easiest is probably to use our **REPL**, by typing `wash up` at the command line. You should see a screen that looks similar to the following:
+There are countless ways to run the actor we just created, and if you went through the [getting started](../../../overview/getting-started) section, you'll have seen some some of them already. Open your browser to view the web dashboard  [http://localhost:4000](http://localhost:4000) to view the dashboard (you may need to modify the port number`4000` if you changed it before launching the wasmCloud host).
+
+At this point, simply click **Start Actor** and then choose _from file_. Open a browser to your current project directory and then navigate to the `build` directory. Here you'll find a `(project)_s.wasm` file, where `(project)` is `hello`, or the name you selected for the project. The `_s` suffix indicates that this WebAssembly module has been _signed_.
+
+A moment later you should see the actor in your web UI as shown in the following screenshot:
+
+TODO screenshot actor started
+
+### Start the web server
+
+We know our new actor needs a web server, so let's start the HTTP server capability provider. To do that, click the **Start Provider** button and enter this OCI URL:  `wasmcloud.azurecr.io/httpserver:0.14.1`. Now we should have both an actor and a provider, and the dashboard should look like this:
 
-![Wash Up](../wash_up.png)
+TODO screenshot actor and provider
 
-Once inside the wash REPL, you can issue this command to start the actor. Make sure the path you supply here is to the _signed_ version of your actor's WebAssembly module.
+### Add a link definition
+
+With both the provider and the actor running, the next step is to _link_ the two. This provides a set of configuration values that is unique for each actor's use of a provider. To change things up slightly, and so you develop some muscle memory with the command line tooling, we'll use the `wash` CLI here. To link your actor, you'll' need the actor's public key. You can get that by clicking on the clipboard icon next to the actor name in the dashboard web UI (it'll be a long string that starts with **M**).
+
+Once you've got the actor's public key, you can enter the following command, replacing `ACTOR_ID` with the key you just put on your clipboard: 
 
 ```
-ctl start actor myactor_s.wasm
+wash ctl link ACTOR_ID wasmcloud.azurecr.io/httpserver:0.14.1 wasmcloud:httpserver port=7000
```

Comment:
This OCI URL needs to be replaced with `VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M` for HTTPSERVER